### PR TITLE
SAK-29239 - Notification says student can resubmit even if assignment is

### PR DESCRIPTION
**Pull request by jonespm**
graded after the accept until date

### CODE REVIEW

**Comment by ottenhoff** [comment]
+1

**Comment by jonespm** [comment]
Thanks for the review. This needed a new method to the api for canSubmit so that it can take a specific user to check against rather than the user in the current session. It was checking for the instructor in this case which wasn't useful in this context. Another issue was that getSubmitterId would return the group if it was a group submission so I had to put that check in for that. I feel like s.getSubmitterIds() should always return one element but it seemed like if it wasn't a group submission it didn't return anything? I need to look at this. I might be a slight simplification since I think checking against one use in the group is enough, otherwise this code is going to get super complex.

**Comment by jonespm** [comment]
That was also why I also changed the subject line (noti.releaseresubmission.subject.content) because the code to append the subject was higher in the call stack and didn't have the same context as the code that created the body, so would require refactoring to have the subject be able to be more specific. I don't think that's as important as the message.

File: assignment/assignment-bundles/resources/assignment.properties [hunk 1]
```diff
@@ -903,8 +903,10 @@ content_review.accepted.types.delimiter=,
 content_review.accepted.types.lparen=(
 content_review.accepted.types.rparen=)
 
-noti.releaseresubmission.subject.content=Email notification for assignment with resubmissions allowed
-noti.releaseresubmission.text=Your submission to assignment "{0}" has been corrected and it allows resubmissions. Please go to {1} to view details.
+noti.releaseresubmission.subject.content=Email notification for assignment
```
Comment (bbailla2) [inline assignment/assignment-bundles/resources/assignment.properties]: Perhaps "Email  notification for assignment release"?

Comment (bbailla2) [inline assignment/assignment-bundles/resources/assignment.properties]: 'Email notification' is redundant in a subject line too. Ideally I'd aim for something like:
"Your assignment has been graded", or such... 'Graded' implies there was a grade, 'corrected' implies there were corrections, 'released' is already defined for when the instructor posts the assignment. Marked perhaps?

**Comment by jonespm** [comment]
Thanks for the feedback @bbailla2 I'm also going to update it, as it looks like s.getSubmittersIds() always has at least one element, so the group check isn't useful.

File: assignment/assignment-bundles/resources/assignment.properties (hunk 1 above)
Comment (jonespm) [inline assignment/assignment-bundles/resources/assignment.properties]: How about "Assignment submission has been corrected"

Comment (bbailla2) [inline assignment/assignment-bundles/resources/assignment.properties]: "Assignment submission has been reviewed" perhaps?

**Comment by jonespm** [comment]
Going to merge this as no further feedback and made the changes

**Pull request merged**